lib/core/plugin.rb

```diff
@@ -115,15 +116,26 @@ def load
 
   # Build out help commands for users to query in chat
   def help_load
-    # case @config['plugin']['type']
-    # when 'script', 'api'
-    #   @help_hash = @config['plugin']['help']
-    # when 'container'
-    #   @help_hash = {}
-    #   @container_info['Labels'].each do |label, desc|
-    #     @help_hash[label] = desc
-    #   end
-    # end
+    @help_list = ''
+    if @container_hash['Labels']
+      @container_hash['Labels'].each do |label, desc|
+        @help_list += '    ' + label + ' : ' + desc + "\n"
```

Comment:
Did the append '<<' operator and interpolation not work here?  If so we might log an minor issue to come back to this at some point.  This is fairly readable, however could likely be a little more sol